src/index.html

```diff
@@ -11,7 +11,129 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="form" >
+      <fieldset class="section">
+        <legend>Personal informatiom:
+        </legend>
+        <label class="input">Surname:
+          <input 
+            type="text" 
+            autocomplete="off" 
+            name="sureName"
+          >
+        </label><br>
+        <label class="input">Name:
+          <input 
+            type="text"
+            autocomplete="off" 
+            name="name" 
+            required 
+          >
+        </label><br> 
+        <label class="input">How old are You?
```

Comment:
Add an empty line between multilane children
```suggestion
        </label>
       
        <label class="input">
          How old are You?
```